onedal/covariance/incremental_covariance.py

```diff
@@ -114,19 +115,24 @@ def finalize_fit(self, queue=None):
         Parameters
         ----------
         queue : dpctl.SyclQueue
-            Not used here, added for API conformance
+            If not None, use this queue for computations.
 
         Returns
         -------
         self : object
             Returns the instance itself.
         """
         params = self._get_onedal_params(self._dtype)
+        if queue is not None:
+            policy = self._get_policy(queue)
+        else:
+            policy = self._get_policy(self._queue)
```

Comment:
I think at finalize_fit call self._queue should be deleted.

Reply:
https://github.com/intel/scikit-learn-intelex/pull/1979#discussion_r1738454650